README.pt.md

```diff
@@ -0,0 +1,106 @@
+e publicadas [no GitHub](https://github.com/SirCmpwn/sway/releases).
+
+## Status
+
+- [Suporte aos recursos do i3](https://github.com/SirCmpwn/sway/issues/2)
+- [Suporte aos recursos IPC](https://github.com/SirCmpwn/sway/issues/98)
+- [Suporte aos recursos do i3bar](https://github.com/SirCmpwn/sway/issues/343)
+- [Suporte aos recursos do i3-gaps](https://github.com/SirCmpwn/sway/issues/307)
+- [Recursos de segurança](https://github.com/SirCmpwn/sway/issues/984)
+
+## Instalação
+
+### A partir de pacotes
+
+Sway está disponível em várias distribuições. Verifique se o pacote "sway" está
+disponível a partir do gerenciador de pacotes da sua distribuição. Caso não esteja,
+procure por informações sobre como instalar o Sway na sua distribuição
+[aqui](https://github.com/SirCmpwn/sway/wiki/Unsupported-packages).
+
+Se você está interessado em manter um pacote do Sway para a sua distribuição,
+visite o canal no IRC ou mande um email para sir@cmpwn.com (*em inglês*).
+
+### A partir do código-fonte
+
+Antes de iniciar a compilação, instale as dependências:
+
+* cmake
+* [wlc](https://github.com/Cloudef/wlc)
+* wayland
+* xwayland
+* libinput >= 1.6.0
+* libcap
+* asciidoc
+* pcre
+* json-c
+* pango
+* cairo
+* gdk-pixbuf2 *
+* pam **
+* imagemagick (capturar imagem com o swaygrab)
+* ffmpeg (capturar vídeo com o swaygrab)
+
+_\*Dependência apenas de swaybar, swaybg, e swaylock_
+
+_\*\*Dependência apenas de swaylock_
+
+Para compilar, execute estes comandos:
+
+    mkdir build
+    cd build
+    cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_SYSCONFDIR=/etc ..
+    make
+    sudo make install
+
+Em sistemas com logind, configure as seguintes capacidades para o arquivo binário:
+
+    sudo setcap cap_sys_ptrace=eip /usr/local/bin/sway
+    sudo setcap cap_sys_tty_config=eip /usr/local/bin/sway
+
+Em sistemas sem logind, ative a *flag* de *suid* do arquivo binário:
+
+    sudo chmod a+s /usr/local/bin/sway
+
+## Configuração
+
+Se você já usa o i3, copie o arquivo de configuração do i3 para `~/.config/sway/config`;
+o Sway lerá o arquivo normalmente. Senão, copie o arquivo de configuração de exemplo
+para `~/.config/sway/config`. É comum esse arquivo estar localizado em
+`/etc/sway/config`. Veja `man 5 sway` para informações sobre configuração.
+
+## Executando
+
+Execute `sway` a partir de um terminal do Linux. Alguns gerenciadores de *display*
+podem funcionar, porém o Sway não procura manter compatibilidade com esses (segundo
+relatos, o gdm funciona bem com o Sway).
+
+## Ajuda em português
+
+No momento, o suporte em português no canal do IRC **não está ativo**. Em caso de problemas,
+use as [*issues*](https://github.com/SirCmpwn/sway/issues/) do Github (*em inglês*).
+
+Essa tradução, apesar de estar comparável com a versão original em inglês, é um
+*trabalho em progresso*. Caso encontre algum erro, visite
+[essa *issue*](https://github.com/SirCmpwn/sway/issues/1318) para mais informações
+e não exite em enviar as correções necessárias.
```

Comment:
The language status summary should go immediately before the releases section.

Reply:
Done.